draft-xyz-rats-psa-endorsements.md

```diff
@@ -53,22 +53,29 @@ informative:
 
 --- abstract
 
-This memo describes a CoMID profile to convey PSA Endorsements.
-
-PSA Endorsements comprise reference values, cryptographic key material and
+PSA Endorsements comprises reference values, cryptographic key material and
```

Comment:
```suggestion
PSA Endorsements comprise reference values, cryptographic key material and
```

Reply:
No comprises is the correct usage, I checked it on the internet yesterday! Leave it like this it is the most correct semantic

Reply:
When you have multiple items then the correct verb to use here is comprises

Reply:
https://grammar.yourdictionary.com/grammar-rules-and-tips/comprise-vs-compose.html

Reply:
I don't understand:  "Endorsements" (plural) needs the third-person plural of the verb "comprise".